ppl/src/main/java/org/opensearch/sql/ppl/utils/PPLQueryDataAnonymizer.java

```diff
@@ -316,5 +329,14 @@ public String visitAlias(Alias node, String context) {
       String expr = node.getDelegated().accept(this, context);
       return StringUtils.format("%s", expr);
     }
+
+    @Override
+    public String visitTrendlineComputation(Trendline.TrendlineComputation node, String context) {
+      final String dataField = node.getDataField().accept(this, context);
+      final String aliasClause = " as " + node.getAlias();
+      final String computationType = node.getComputationType().name().toLowerCase(Locale.ROOT);
```

Comment:
In `Trendline.java` you use uppercase. Just to confirm that there is no conflict.

Reply:
Yes this is intentional. It's upper-cased there so that it can be passed to Enum.valueOf() correctly, but lower-cased here which seems to be the standard for writing PPL.